keanu-project/src/main/java/io/improbable/keanu/vertices/dbl/probabilistic/BetaVertex.java

```diff
@@ -108,6 +110,30 @@ public double logProb(DoubleTensor value) {
         return logPdfs.sum();
     }
 
+    @Override
+    public LogProbGraph logProbGraph() {
```

Comment:
I'm not convinced that a `LogProbGraph` should be comprised of `Vertex`s. A Log Prob graph must be comprised only of deterministic nodes, but currently it's possible to include probabilistic vertices in a `LogProbGraph`.

Reply:
Will discuss offline.